tests/scripts/docker/docs.sh

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env bash
 set -e
 
+cd tests
+
 docker build -t "lint-docs" -f ./DockerfileDocs .
-docker run -i "lint-docs"
+docker run -e LOCAL_USER_ID=`id -u $USER` -v "$(dirname $(pwd))":/app -i "lint-docs"
```

Comment:
Since this is running bash, which what you have here is perfectly correct, you could also simplify it by just using `-e LOCAL_USER_ID=$UID`.

One style comment, the backticks have been deprecated for some time in favor of `$()` in [POSIX shells](http://pubs.opengroup.org/onlinepubs/9699919799/xrat/V4_xcu_chap02.html#tag_23_02_06_03). It's recommended to use `$( )` instead.